### PR TITLE
[WIP] fix(issuer): make header row mandatory on whitelist

### PR DESCRIPTION
**Pull request by sajclarke**
**Please make sure the following boxes are checked before submitting your Pull Request:**

- [x] I linked the issues related to this PR in its description (if any).
- [ ] If this PR adds new code that is not going to change in the near future, it includes unit tests to cover it.

### CODE REVIEW

File: packages/polymath-issuer/src/utils/parsers/index.js [hunk 1]
```diff
@@ -83,68 +83,89 @@ const checkColumnCount = data => {
   return errorMsg;
 };
 
+const checkCSVHeaders = data => {
+  // Split the input into lines
+  let rows = data.split('\n');
+  const header_row = rows[0].split(',');
+  const default_headers =
+    'Address,Sale Lockup,Purchase Lockup,KYC/AML Expiry,Can Buy From STO,Exempt From % Ownership';
```
Comment (josepmc) [inline packages/polymath-issuer/src/utils/parsers/index.js]: Is Exempt from % ownership a default header? The user may not have the max % ownership enabled.

File: packages/polymath-issuer/src/utils/parsers/index.js [hunk 2]
```diff
@@ -83,68 +83,89 @@ const checkColumnCount = data => {
   return errorMsg;
 };
 
+const checkCSVHeaders = data => {
```
Comment (josepmc) [inline packages/polymath-issuer/src/utils/parsers/index.js]: What's the behaviour with the isAccredited and nonAccreditedLimit fields?

File: packages/polymath-issuer/src/utils/parsers/index.js [hunk 3]
```diff
@@ -83,68 +83,89 @@ const checkColumnCount = data => {
...
+            return false;
+          }
+          if (value.split('/').length === 3) {
+            const [rawMonth, day, rawYear] = value.split('/');
+            // Months are 0-based
+            const month = parseInt(rawMonth, 10) - 1;
+            let year = parseInt(rawYear, 10);
+
+            // Support for incomplete years
+            if (year < 2000) {
```
Comment (josepmc) [inline packages/polymath-issuer/src/utils/parsers/index.js]: Are we sure we want to have something like this? The issuer inputs 31/12/1999 and then we bump it to 31/12/3999?

File: packages/polymath-issuer/src/utils/parsers/index.js (hunk 1 above)
Comment (sajclarke) [reply packages/polymath-issuer/src/utils/parsers/index.js]: Yes it's default. It'll be empty if not enabled.

File: packages/polymath-issuer/src/utils/parsers/index.js (hunk 3 above)
Comment (sajclarke) [reply packages/polymath-issuer/src/utils/parsers/index.js]: Good point. The intention is probably to capture if the value is between 1 and 100 then adjust it accordingly. Would be easier to invalidate the year instead I think. Thoughts @RafaelVidaurre ?

File: packages/polymath-issuer/src/utils/parsers/index.js (hunk 2 above)
Comment (sajclarke) [reply packages/polymath-issuer/src/utils/parsers/index.js]: I think those fields are optional (if usdsto is attached)

File: packages/polymath-issuer/src/utils/parsers/index.js (hunk 3 above)
Comment (RafaelVidaurre) [reply packages/polymath-issuer/src/utils/parsers/index.js]: As discussed on slack, yup, let's fix this.

**Pull request merged**